Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -383,28 +393,34 @@ void GameListCtrl::RefreshList()
   if (Core::GetState() != Core::State::Uninitialized)
     return;
 
+  // Use a newly loaded title database (it might have gotten updated)
+  Core::TitleDatabase title_database;
+
+  m_shown_names.clear();
   m_shown_files.clear();
   {
     std::unique_lock<std::mutex> lk(m_cache_mutex);
-    m_cache.ForEach([this](const std::shared_ptr<const UICommon::GameFile>& game_file) {
-      if (ShouldDisplayGameListItem(*game_file))
-        m_shown_files.push_back(game_file);
-    });
+    m_cache.ForEach(
+        [this, title_database](const std::shared_ptr<const UICommon::GameFile>& game_file) {
```

Comment:
Did you intend to take the title database by const ref?

Reply:
Or, well, it's not possible to use a const ref in a lambda capture as far as I know... But I do want to take a ref.

Reply:
IIRC everything you capture into a lambda is implicitly const, unless you add mutable before the `{`.

Reply:
Hmm, just checked and it seems like I'm misremembering. However if you make title_database const it should have the same effect.